beacon-chain/blockchain/forkchoice/process_block.go

```diff
@@ -135,13 +135,6 @@ func (s *Store) OnBlock(ctx context.Context, b *ethpb.BeaconBlock) error {
 	if helpers.IsEpochStart(postState.Slot) {
 		logEpochData(postState)
 		reportEpochMetrics(postState)
-
-		// Update committee shuffled indices at the end of every epoch
```

Comment:
Why remove this?

Reply:
The seed->committees updates upon cache miss, not at predetermined boundary slot.

This design was inefficient and no longer needed

Reply:
Why is it inefficient? Doesn't it make sense to prepopulate this cache before it's used?

Please update PR description, this change wasn't mentioned.